sdk/nodejs/dynamic/index.ts

```diff
@@ -14,9 +14,124 @@
 
 import { Inputs, Input, Output } from "../output";
 import * as resource from "../resource";
+import { getProject } from "../metadata";
+import * as pulumiConfig from "../config";
 import * as settings from "../runtime/settings";
 import * as serializeClosure from "../runtime/closure/serializeClosure";
 
+export type ConfigType = Pick<pulumiConfig.Config,
+    "get" | "getBoolean" | "getNumber" | "getObject">;
```

Comment:
Do we need the `require*` methods too?

Reply:
Yep! forgot them

Reply:
Actually, this type is not really necessary. I was initially trying to have something clever to avoid copying the method signatures, but that didn't work out.